library/src/main/java/com/google/android/exoplayer2/extractor/flv/VideoTagPayloadReader.java

```diff
@@ -92,7 +93,9 @@ protected void parsePayload(ParsableByteArray data, long timeUs) throws ParserEx
           Format.NO_VALUE, Format.NO_VALUE, avcConfig.width, avcConfig.height, Format.NO_VALUE,
           avcConfig.initializationData, Format.NO_VALUE, avcConfig.pixelWidthAspectRatio, null);
       output.format(format);
-      hasOutputFormat = true;
+      if (listener != null) {
```

Comment:
Are these packet types frequent on some streams?  Should we check if format matches and not call the listener if it didn't actually change?